### PR TITLE
[AudioToolbox] Update bindings to Xcode 12.5 Beta 1 to Beta 3

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/AudioUnit/AudioComponent.cs [hunk 1]
```diff
@@ -448,6 +448,10 @@ public static class AudioComponentValidationParameter {
 //		#define kAudioComponentValidationParameter_ForceValidation		 "ForceValidation"
 		public static NSString ForceValidation = new NSString ("ForceValidation");
 
+//		#define kAudioComponentValidationParameter_LoadOutOfProcess		 "LoadOutOfProcess"
+		[iOS (14,5), Mac (11,3)]
+		public static NSString LoadOutOfProcess = new NSString ("LoadOutOfProcess");
```
Comment (spouliot) [inline src/AudioUnit/AudioComponent.cs:453]: it's added in the tvOS assembly, should it be ?
if so there should be an availability attribute

File: src/AudioUnit/AudioUnit.cs [hunk 2]
```diff
@@ -1802,7 +1804,9 @@ public enum AURenderEventType : byte
 	[iOS (9,0), Mac (10,11)]
 	public enum AudioComponentInstantiationOptions : uint {
 		OutOfProcess = 1,
-		InProcess = 2
+		InProcess = 2,
+		[iOS (14,5), Mac (11,3)]
+		LoadedRemotely = 1u << 31,
```
Comment (spouliot) [inline src/AudioUnit/AudioUnit.cs:1809]: it's added in the tvOS assembly, should it be ?
if so there should be an availability attribute or a `[NoTV]`

**Review by spouliot — CHANGES_REQUESTED** [review]

File: src/AudioUnit/AudioComponent.cs [hunk 3]
```diff
@@ -448,6 +448,10 @@ public static class AudioComponentValidationParameter {
 //		#define kAudioComponentValidationParameter_ForceValidation		 "ForceValidation"
 		public static NSString ForceValidation = new NSString ("ForceValidation");
 
+//		#define kAudioComponentValidationParameter_LoadOutOfProcess		 "LoadOutOfProcess"
+		[iOS (14,5), Mac (11,3)]
```
Comment (dalexsoto) [inline src/AudioUnit/AudioComponent.cs:452]:
```suggestion
		[iOS (14,5), TV (14,5), Mac (11,3)]
```

File: src/AudioUnit/AudioUnit.cs [hunk 4]
```diff
@@ -1169,6 +1169,8 @@ enum AudioUnitPropertyIDType { // UInt32 AudioUnitPropertyID
 		ParametersForOverview = 57,
 		[iOS (10,0), Mac (10,12)]
 		SupportsMpe = 58,
+		[iOS (14,5), Mac (11,3)]
```
Comment (dalexsoto) [inline src/AudioUnit/AudioUnit.cs:1172]:
```suggestion
		[iOS (14,5), TV (14,5), Mac (11,3)]
```

File: src/AudioUnit/AudioUnit.cs [hunk 5]
```diff
@@ -1802,7 +1804,9 @@ public enum AURenderEventType : byte
 	[iOS (9,0), Mac (10,11)]
 	public enum AudioComponentInstantiationOptions : uint {
 		OutOfProcess = 1,
-		InProcess = 2
+		InProcess = 2,
+		[iOS (14,5), Mac (11,3)]
```
Comment (dalexsoto) [inline src/AudioUnit/AudioUnit.cs:1808]:
```suggestion
		[iOS (14,5), TV (14,5), Mac (11,3)]
```

**Review by spouliot — APPROVED** [review]

**Pull request merged**